en/plugins/zapier.md

```diff
@@ -4,12 +4,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
 
 ## Requirements
 
-1. Mautic version 2.9.0 or newer with SSL - basic authentication requires HTTPS to be secure.
-3. Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
+Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
 
 ## Installation
 
-There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic. This Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
+There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic.
+
+### Oauth2 Authentication for Zapier 3.x
+
+New version of app improve authentication by Oauth2 protocol.
```

Comment:
This new version of the app improves authentication by implementing the Oauth2 protocol